features/update/async_accepted/feature.go

```diff
@@ -0,0 +1,134 @@
+package async_accepted
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	enumspb "go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/serviceerror"
+	updatepb "go.temporal.io/api/update/v1"
+	"go.temporal.io/features/features/update/updateutil"
+	"go.temporal.io/features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+)
+
+type UpdateDisposition int
+
+const (
+	theUpdate       = "theUpdate"
+	theUpdateResult = 123
+	shutdownSignal  = "shutdown_signal"
+
+	succeed       UpdateDisposition = 0
+	failWithError UpdateDisposition = 1
+	failWithPanic UpdateDisposition = 2
+)
+
+var Feature = harness.Feature{
+	Workflows: Workflow,
+	Execute: func(ctx context.Context, runner *harness.Runner) (client.WorkflowRun, error) {
+		if reason := updateutil.CheckServerSupportsAsyncAcceptedUpdate(ctx, runner.Client); reason != "" {
+			return nil, runner.Skip(reason)
+		}
+		run, err := runner.ExecuteDefault(ctx)
+		if err != nil {
+			return nil, err
+		}
+
+		updateCounter := 0
+		requestedSleep := 2 * time.Second
+		sendAnAsyncUpdate := func(
+			ctx context.Context,
+			disp UpdateDisposition,
+		) (client.WorkflowUpdateHandle, error) {
+			updateCounter++
+			return runner.Client.UpdateWorkflowWithOptions(
+				ctx,
+				&client.UpdateWorkflowWithOptionsRequest{
+					UpdateID:   fmt.Sprintf("update:%v", updateCounter),
+					WorkflowID: run.GetID(),
+					RunID:      run.GetRunID(),
+					UpdateName: theUpdate,
+					Args:       []interface{}{requestedSleep, disp},
+					WaitPolicy: &updatepb.WaitPolicy{
+						LifecycleStage: enumspb.UPDATE_WORKFLOW_EXECUTION_LIFECYCLE_STAGE_ACCEPTED,
+					},
+				},
+			)
+		}
+
+		start := time.Now()
+		originalHandle, err := sendAnAsyncUpdate(ctx, succeed)
```

Comment:
If possible, again because I and others are kinda reading this as a how-to currently, can you add a simple comment above here and before `sendAnAsyncUpdate` in sections below on the situation you're testing. E.g.

```suggestion
		// Simple accepted update should return quickly not wait for completion
		originalHandle, err := sendAnAsyncUpdate(ctx, succeed)
```